README.md

```diff
@@ -54,91 +65,110 @@ Okay, now you know why to document let's define some rules that everyone has to
 - How to handle the most common errors like ```package not found``` and others
 
-##### Simulation Engineer
-[Gazebo]()
-[Unity]()
+### Simulation Engineer
 
+1. [Gazebo](http://gazebosim.org/)
+2. [Unity](https://unity.com/)
 
 ---
-## Management ::   
-##### Roles
-##### Treasurer
+
+## Management
+
+### Roles
+
+### Treasurer
+
 1. What's the procedure for a change in the signature at the bank for the club bank account
 2. Procedure for transferring of Univesity funds (The list of items to be prepared to sum up the money?)
 3. The format in which the students' names and contacts to be maintained along with their application fees
 4. Format to maintain the credit and debit of funds in the club (Including club fees, alumni funds, crowdfunding, workshop incentives)
 
-##### Event Manager
-##### Video/Photographer
+### Event Manager
+
+### Video/Photographer
+
 1. Usage of video editing software
 2. Usage of photo editing hardware
-4. Preserve all raw images/footages used for making videos
-5. Upload the final material on YouTube
-6. any unfinished projects must be handed over to the Coordinator/Faculty Advisor using a Pen Drive.
-7. All those that covers point  4. must be stored in a secure place and handed over to Coordinator/Faculty Advisor using a Pen Drive/Hard Disk.
+3. Preserve all raw images/footages used for making videos
+4. Upload the final material on YouTube
+5. any unfinished projects must be handed over to the Coordinator/Faculty Advisor using a Pen Drive.
+6. All those that covers point  4. must be stored in a secure place and handed over to Coordinator/Faculty Advisor using a Pen Drive/Hard Disk.
+
+### Creative Coordinator
 
-##### Creative Coordinator
 1. Usage of poster/banner making software
 2. All images used in the posters must be preserved (except easily found logos; internet pictures and backgrounds)
 3. Preserve all the raw photoshop edited files
 4. Hand over items from 3. to the Coordinator/Faculty Advisor using a Pen Drive.
 
-##### Workshop Coordinator
+### Workshop Coordinator
+
 1. Format of the letter sent to all the previously requested schools/universities for workshops
 2. The package proposed to be held - The course covered for 2 days/3 days (Mechanical, Electrical/Electronics, and Computer); Kit taken along; the per head charge taken
 3. A complete report of the conducted workshop - Place, Institute held in, attendance of students, how many club members invited, total revenue generated(show by subtracting the total travel & accommodation cost)
 4. Any upcoming/incomplete request or possible places/institutes/colleges who showed any amount of interest for conduction of workshop but couldn't be followed up because of preoccupations, awkward timing, non-negotiable fees or finish of tenure - mention all the conversations and outcomes
 
-##### Social Media Coordinator
+### Social Media Coordinator
+
 [Role](https://github.com/auvzhcet/Documentation/blob/master/Social_Media_Manager.md)
```

Comment:
Change this link.

Reply:
can you please give me the required link?

Reply:
Look where `social_media_manager.md` file is.

Reply:
I got the file but no link is provided there

Reply:
ok